### DISCUSSION TITLE
C drive is full. chia.exe start up stuck at full node.

### DISCUSSION THREAD

**Discussion by greenfrogy8888** [Chia-Network/chia-blockchain#10845] [Support]
Hello all,
My C drive is full due to .chia folder. Per advice on the internet I
found, I moved the db & wallet folder to E drive. Also edit
config.yaml file to point to
E:/chia_blockchain/db/blockchain_v1_CHALLENGE.splite from
db/blockchain_v1_CHALLENGE.sqlite
I been trying to troubleshoot myself for months now.
I follow this weblink on how to move chia db.
https://spacefarmers.io/wiki/guides/farming/movedb
Now whenever I launch chia version 1.3.1 , it will stuck at full node.
Please advise me how to resolve this and what is the proper way of
moving .chia from C drive to other drive and continue farming.
I also attached my log for review.

Thank you for your help in advance.
[debug.log](https://github.com/Chia-Network/chia-blockchain/files/8326149/debug.log)

**Comment by Gooseboy2234** [comment]
have you upgraded the DB version? Are you using windows file compression on the files? What is your free space? Is it literary 0 bytes left?

**Reply by greenfrogy8888** [reply]
I don't know what version of DB I have. Can you tell me how to check or convert it ? Is it necessary since I am on ver 1.3.1
Since your note, here is what I done. I found other sites on internet saying to setup hard link to my .chia folder. So I moved .chia & chia-blockchain folder to E drive and made a hardlink using mklink /J. Corrected my config.yaml to E:/.chia or E:/chia-blockchain depending on the original settings in config.yaml.  On full node menu, it would say syncing 0/1,748,343 and the number slowly increasing to like 0/1,748,556. In plot, it would say syncing but when you mouse hover over it. It would say wait for syncronization.

**Reply by greenfrogy8888** [reply]
I don't use windows file compression. My C drive is only 182GB with 44.1GB free. That free 44.1GB was taken by chia db. I follow googled instructions to move .chia & chia-blockchain folders to E: drive. Then I use mklink /J command to make a hard link to E:/ . My config.yaml is pointing to E:/.chia/mainnet/db... & E:/chia-blockchain... On full node menu my sync is in orange or amber saying syncing 0/1,733,3... This number does slowly increasing. plots menu indicate syncing but when hover over it says wait for syncronization. What is the official way or proper way of making non-c drive as chia drive ? How to check if my DB version is current or how to upgrade ? Attached debug log
[debuglog.txt](https://github.com/Chia-Network/chia-blockchain/files/8356503/debuglog.txt)

**Reply by Gooseboy2234** [reply]
ok, do these two steps
but keep in mind, these will only HELP the problem and you might need to get a bigger drive

uninstall chia
delete .chia and .chia-keys
reinstall chia
launch chia
wait for 1 hr [for the chia to sync a few thousand blocks]
close chia
perform DB upgrade:
https://www.youtube.com/watch?v=3HVcj4G4y_8&t=179s
activate windows compression
go to .chia/database
open priorities menu for db and wallet
click the advanced button on both menu's
 pick the checkmark "Compress contents to save disk space" on both
pick ok on both
click apply on both [You might need to wait a hour or two after doing this]
click ok on both
reopen chia
sync!!! [You're done]!!!
